code/lib/cli/src/automigrate/fixes/eslint-plugin.ts

```diff
@@ -79,26 +81,38 @@ export const eslintPlugin: Fix<EslintPluginRunOptions> = {
     if (!dryRun) packageManager.addDependencies({ installAsDevDependencies: true }, deps);
 
     if (!dryRun && unsupportedExtension) {
-      throw new Error(dedent`
+      logger.info(dedent`
           ⚠️ The plugin was successfuly installed but failed to configure.
           
-          Found an .eslintrc config file with an unsupported automigration format: ${unsupportedExtension}.
-          Supported formats for automigration are: ${SUPPORTED_ESLINT_EXTENSIONS.join(', ')}.
+          Found an eslint config file with an unsupported automigration format: .eslintrc.${unsupportedExtension}.
+          The supported formats for this automigration are: ${SUPPORTED_ESLINT_EXTENSIONS.join(
+            ', '
+          )}.
 
           Please refer to https://github.com/storybookjs/eslint-plugin-storybook#usage to finish setting up the plugin manually.
       `);
     }
 
-    const eslint = await readConfig(eslintFile);
-    logger.info(`✅ Configuring eslint rules in ${eslint.fileName}`);
-
+    logger.info(`✅ Adding Storybook plugin to ${eslintFile}`);
```

Comment:
Maybe I'm misunderstanding here, but with these changes (not throwing an error anymore), wouldn't we end up with some confusing logging when users are using an unsupported extension?:

```bash
⚠️ The plugin was successfuly installed but failed to configure......
✅ Adding Storybook plugin to undefined
```

What's the reasoning behind this?

Reply:
Sorry, there was supposed to be a `return` right after that log. Thanks for catching that!

The output will be:
```
⚠️ The plugin was successfuly installed but failed to configure.

Found an eslint config file with an unsupported automigration format: .eslintrc.yml.
The supported formats for this automigration are: js, cjs, json.

Please refer to https://github.com/storybookjs/eslint-plugin-storybook#usage to finish setting up the plugin manually.
✅ ran eslintPlugin migration
```

There is no way for `eslintFile` to be undefined by the time it reaches that `run` function

Reply:
The reasoning behind changing from an error to just a log is that the migration didn't really error, so that's more like a warning